crates/topos-api/proto/topos/tce/v1/double_echo.proto

```diff
@@ -30,3 +30,7 @@ message DoubleEchoRequest {
     Ready ready = 3;
   }
 }
+
+message Batch {
```

Comment:
I'm wondering if creating a new proto file just for GossipSub related message could be beneficial, `Batch` isn't really part of the `double_echo` it is only an Envelope for the GossipSub protocol.

We could have something like:

```protobuf
file: `proto/topos/tce/v1/gossipsub.proto`

package topos.tce.v1;

message Batch {
	repeated bytes messages = 1;
}
```

Reply:
@Freyskeyd Problem I see with Batch having `repeated bytes messages` is how to de-serialize them. 

If we have `Batch` having other proto structure as its payload, we encode them at one point. If we first encode unknown number of multiple double echo messages to byte array, and then encode Batch, when we decode Batch on the other size we don't know how to decode `messages` and what is the payload meaning

Reply:
@Freyskeyd Still it can not be just byte array. Because from one byte array, you don't know how big is the vector of byte array messages